app/cdap/components/Replicator/Create/Content/PluginConfig/index.tsx

```diff
@@ -182,11 +196,10 @@ const PluginConfigView: React.FC<IPluginConfigProps> = ({
       processedConfigurationGroup.configurationGroups,
       pluginWidget,
       pluginProperties,
```

Comment:
@elfenheart is it possible for plugin properties to change here? its based on plugin info, which is a prop. Unsure if i need to make a ref for plugin info as well?